configure.ac

```diff
@@ -131,8 +131,44 @@ AC_ARG_WITH([docs_only],
                        [Compile only the docs and not the rest of UCX. [default=NO]]),
         ,[:],[with_docs_only=no])
 
+AC_DEFUN([UCX_ARG_DOCS_ENABLE],
```

Comment:
i though there are already such flags in doxygen..
so i don't it's needed for this PR. can add those as separate PR

Reply:
removed from this PR